EcalTiming/test/ecalTime_fromAlcaStream_cfg.py

```diff
@@ -57,9 +74,18 @@
     process.load('Configuration/StandardSequences/Reconstruction_cff')
     process.recoSequence = cms.Sequence(process.calolocalreco )#+ process.egammaCosmics)
 
-process.load('PhiSym.EcalCalibAlgos.ecalPhiSymLocarecoWeights_cff')
+#process.load('PhiSym.EcalCalibAlgos.ecalPhiSymLocarecoWeights_cff')
+#process.load('RecoLocalCalo.Configuration.ecalLocalRecoSequence_cff')
 process.load('Configuration.StandardSequences.EndOfProcess_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+process.load('EcalTiming.EcalTiming.ecalLocalRecoSequenceAlCaStream_cff')
```

Comment:
if you load the reconstruction file in the stream definition/package, this will be automatically right